mocha/common.js

```diff
@@ -0,0 +1,8 @@
+global.React = require('react');
+global.sinon = require('sinon');
+global.chai = require("chai");
+global.should = chai.should();
```

Comment:
Is there a reason to stick with one? I guess our usage in other projects seem to flip between the two styles wherever it is most readable.